docs/usage.md

```diff
@@ -63,31 +63,64 @@ for site_id, pos in enumerate(positions):
-for inference as long as the indel does not overlap with other variants, only 2
+for inference as long as the indel does not overlap with other variants, only two
 alleles exist, and the ancestral state is known.
 :::
 
-### VariantData and ancestral alleles
+(sec_usage_toy_example_variant_data)=
 
-We wish to infer a genealogy that could have given rise to this data set. To run _tsinfer_
-we wrap the `.vcz` file in a {class}`tsinfer.VariantData` object. This requires an 
-*ancestral state* to be specified for each site; there are
-many methods for calculating these: details are outside the scope of this manual, but we
-have started a [discussion topic](https://github.com/tskit-dev/tsinfer/discussions/523)
+
+### VariantData
+
+_Tsinfer_ produces a genealogy that could have given rise to the data set above,
+based on the sites that vary between the samples. To provide extra information
+to the algorithm, you must wrap the .vcz file in a lightweight
+{class}`tsinfer.VariantData` object, using syntax like:
+
+```{code} python
+vdata = tsinfer.VariantData("file.vcz", ancestral_state=***, ...)
+```
+
+#### Ancestral states
+
+Importantly, the {class}`~tsinfer.VariantData` object requires an
+*ancestral state* to be provided for each site used in inference.
+There are many methods for determing ancestral states: details are outside
+the scope of this manual, but we have started a
+[discussion topic](https://github.com/tskit-dev/tsinfer/discussions/523)
 on this issue to provide some recommendations.
 
-Sometimes VCF files will contain the
-ancestral state in the "AA" ("ancestral allele") info field, in which case it will be encoded
-in the `variant_AA` field of the .vcz file. It's also possible to provide a numpy array
-of ancestral alleles, of the same length as the number of selected variants. If you have a string
-of the ancestral states (e.g. from FASTA) the {meth}`add_ancestral_state_array`
-method can be used to convert and save this to the VCF Zarr dataset (under the name
-`ancestral_state`). Note that this method assumes that the string uses zero-based
-indexing, so that the first letter corresponds to a site at position 0. If,
-as is typical, the first letter in the string denotes the ancestral state of
-the site at position 1 in the .vcz file, you should prepend an "X" to the string.
-Alleles that are not in the list of alleles for their respective site are
-treated as unknown and not used for inference (with a warning given).
+Ancestral states can be specified in several ways:
+
+* Using an existing array in the .vcz file. Some VCF files already contain an ancestral
+  state, e.g. in the `AA` ("ancestral allele") `info` field. In this case the `variant_AA`
+  field of the .vcz file can be specified, as follows.
+  ```{code}
+  vdata = tsinfer.VariantData("file.vcz", ancestral_state="variant_AA")
+  ```
+  A worked example is shown in the {ref}`real data example<sec_usage_read_vcf_inference>`
+  later on this page
```

Comment:
```suggestion
  later on this page.
```

Reply:
Nice spot. And it does indeed look like a nit egg.

Added in latest commit